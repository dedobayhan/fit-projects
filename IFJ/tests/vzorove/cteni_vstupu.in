12 516.698765432178632467963657654321549876513213249876541321 "dnes"
12     


516.6"dnes">>>text na vstupu, ktery uz ale nebude precten.
ptate se proc?
nu, ridici kod obsahuje postupne cteni hodnot typu: int double string int double string
neni v tom zadna veda

