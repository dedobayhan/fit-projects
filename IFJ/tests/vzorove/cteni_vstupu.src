var int i, double d, string s;

void main()
{
  i := readint();
  d := readdouble();
  s := readstring();
  printint(i);
  printdouble(d);
  printstring(s);
  
  i := readint();
  d := readdouble();
  s := readstring();
  printint(i);
  printdouble(d);
  printstring(s);
  
}
