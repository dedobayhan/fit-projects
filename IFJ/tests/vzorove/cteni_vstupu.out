12516.699dnes12516.6dnes